packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec

```diff
@@ -48,16 +48,13 @@ Provides: foreman-plugin-%{plugin_name} = %{version}
 # start package.json devDependencies BuildRequires
 BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
 BuildRequires: %{?scl_prefix}npm(@babel/core) < 7.8.0
-BuildRequires: %{?scl_prefix}npm(@redhat-cloud-services/frontend-components) >= 2.5.0
-BuildRequires: %{?scl_prefix}npm(@redhat-cloud-services/frontend-components) < 3.0.0
```

Comment:
this breaks it.

```
ERROR in /builddir/build/BUILDROOT/rubygem-foreman_rh_cloud-5.0.29-1.fm3_2.el8.noarch/usr/share/gems/gems/foreman_rh_cloud-5.0.29/webpack/InsightsCloudSync/Components/InsightsTable/table.scss
Module build failed: ModuleNotFoundError: Module not found: Error: Can't resolve '@redhat-cloud-services/frontend-components/index.css' in '/builddir/build/BUILDROOT/rubygem-foreman_rh_cloud-5.0.29-1.fm3_2.el8.noarch/usr/share/gems/gems/foreman_rh_cloud-5.0.29/webpack/InsightsCloudSync/Components/InsightsTable'
```

Reply:
It's strange, I do see `./packages/foreman/nodejs-redhat-cloud-services-frontend-components/nodejs-redhat-cloud-services-frontend-components-2.5.0`. And the version looks correct.

Reply:
Sure, but you removed the BuildRequires, so nothing pulls it in.